src/views/splash.html

```diff
@@ -0,0 +1,134 @@
+        <link rel="icon" href="{{ asset('images/hc-2018/favicons/favicon.png') }}" sizes="512x512" type="image/png">
+        <link rel="icon" href="{{ asset('images/hc-2018/favicons/favicon.ico') }}" sizes="16x16 24x24 32x32 48x48 64x64 96x96 128x128 256x256" type="image/vnd.microsoft.icon">
+        <link rel="icon" href="{{ asset('images/hc-2018/favicons/favicon.icns') }}" sizes="16x16 24x24 32x32 48x48 64x64 96x96 128x128 256x256 512x512">
+        <link rel="icon" href="{{ asset('images/hc-2018/favicons/favicon.svg') }}" sizes="any" type="image/svg+xml">
+        <meta name=description content="Get hyped for Hack Cambridge 2018">
+        <meta name=keywords content="hack,Cambridge,ternary,sponsorship,hackathon">
+        <meta name=author content="Hack Cambridge Team">
+        <meta name=viewport content="width=device-width, initial-scale=1.0">
+    </head>
+    <body>
+        <main>
+            <section id="splash-intro">
+                <div class="container width">
+                    <div class="seven unit row">
+                        <div class="six unit h">
+                            <object id="splash-hack-cambridge-logo-object" data="{{ asset('images/hc-2018/logo.svg') }}"></object>
+                        </div>
+                        <div id="splash-intro-content" class="seven unit h row">
+                            <div class="five unit h">
+                                <div class="container">
+                                    <div class="two unit row">
+                                        <div class="five unit h">
+                                            <div>
+                                                <h2 id="sponsorship-hack-cambridge-wordmark">Hack<br>Cambridge<br><span class="pixel">Ternary</span></h2>
+                                            </div>
+                                        </div>
+                                    </div>
+                                    <div class="two unit row">
+                                        <div class="five unit h">
+                                            <h3>
+                                                Not twice, but three times.<br>Not 2D, but 3D.<br>Not binary, but ternary.
+                                            </h3>
+                                        </div>
+                                    </div>
+                                    <div class="two unit row">
+                                        <div class="five unit h">
+                                            <div id="splash-intro-text">
+                                                <p>
+                                                    In January 2018, Cambridge University’s annual hackathon will yet again bring together 300 top Computer Scientists and Engineers from the best universities worldwide.<br>For 24 hours they will build, break and innovate to produce projects that push the boundaries of technology.
```

Comment:
Nit: break this line — generally it makes sense to break after roughly 150 characters